xml/obs_notifications.xml

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE chapter
+[
+   <!ENTITY % entities SYSTEM "entity-decl.ent">
+   %entities;
+]>
+
+<chapter version="5.1" xml:id="cha.obs.notifications"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+ <title>Notifications</title>
+
+ <note>
+  <title>
+   The feature described in this chapter is not included yet in any OBS
+   official release.
+  </title>
+
+  <para>
+   That means this feature is deployed in the
```

Comment:
@eduardoj how about mentioning its only available through our Unstable project which we deploy on `build.opensuse.org`

Reply:
Fine. Applied.